politeiawww/api/v1/v1.go

```diff
@@ -136,6 +136,11 @@ const (
 	PropStatusCensored    PropStatusT = 3 // Proposal has been censored
 	PropStatusPublic      PropStatusT = 4 // Proposal is publicly visible
 	PropStatusLocked      PropStatusT = 6 // Proposal is locked, NOT IMPLEMENTED
+
+	// Proposal voting status
+	ProposalVotingActive     PropVotingStatus = 0 // proposal on voting
```

Comment:
See comment in api doc on which statuses you need. Also make sure you create a map for these statuses that maps between the error and a human readable string.

Don't use Voting, use Vote (again, see example in api.md comments)